src/interfaces/ICryptoGeneratorService.ts

```
@@ -0,0 +1,3 @@
+export interface ICryptoGeneratorService {
+  // Define methods relevant to crypto generation if used in webhook.controller.ts
+}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)